src/topic_based_system.cpp

```diff
@@ -91,6 +91,7 @@ CallbackReturn TopicBasedSystem::on_init(const hardware_interface::HardwareInfo&
         // Check the initial_value param is used
         if (!interface.initial_value.empty())
         {
+          joint_states_[index][i] = std::stod(interface.initial_value);
```

Comment:
By setting this can it cause the connected robot to jump to the initial position when started?